spinnaker_service/src/main/java/revature/projectFactory/spinnaker/App.java

```diff
@@ -11,14 +11,19 @@
 import revature.projectFactory.spinnaker.connectionUtils.ConnectionConstants;
 import revature.projectFactory.spinnaker.servlets.pipelineServlet;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  * Hello world!
  *
  */
 public class App 
 {
+
     public static void main( String[] args )
     {
+        final Logger log = LoggerFactory.getLogger(App.class);
```

Comment:
This works, but for future reference you want the logger creation line to be _outside_ any methods.  You have it within the main method here.  You want the logger to be available to the entire class rather than just within the method.

Reply:
Done, thank you!!